examples/00-echo/src/main.rs

```diff
@@ -0,0 +1,139 @@
+//! A simple echo application with Fluvio, to demonstrate producing and consuming messages
+//!
+//! Fluvio is a streaming platform with a simple API for quick development.
+//! This example demonstrates how to perform two key streaming operations:
+//! producing and consuming messages.
+//!
+//! # Installation
+//!
+//! Before running this example, you need to make sure you have Fluvio installed.
+//! If you haven't yet, visit our [installation page], then come back here.
+//!
+//! Once you have the Fluvio CLI and you have a Fluvio cluster available
+//! (either through minikube or Fluvio Cloud), you're ready to get started.
+//!
+//! [installation page]: https://nightly.fluvio.io/docs/kubernetes/install/
```

Comment:
@ajhunyady so this is already become issue.  not show stopper for now but it will become issue when release or new releases.

Reply:
so, how do we solve it?

Reply:
maybe for now, just stick with stable URL.    may not be problem since example doc will not be published

Reply:
It doesn't look like an obvious fix to me. The example should always point to latest. Only reference a target release if the API changes. The link will be broken until we release nightly... but that should be ok.